src/Rule/Count.php

```diff
@@ -158,11 +156,4 @@ public function getHandler(): string
     {
         return CountHandler::class;
     }
-
-    public function afterInitAttribute(object $object, int $target): void
```

Comment:
Is it alright?

Reply:
Yes. Now object itself passed to validation when rule applied to whole object.